safep/moving_wall_TI.py

```
@@ -0,0 +1,252 @@
+        Returns:
+            None
+
+        Side Effects:
+            Creates the force column
+        """
+        if "wall_position" not in self.columns:
+            self.get_wall_position(config)
+        k = config["spring"]
+        mask = self.DBC > self.wall_position
+        self["force"] = 0.0
+        compute_force = lambda sample: k * (sample["wall_position"] - sample["DBC"])
```

Ruff (0.16.0)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `compute_force` as a `def`

(E731)